slides/welcome.md

```diff
@@ -0,0 +1,50 @@
+---
+title:  Course Practicalities
+date:   2023-06-20
+lang:   en
+---
+
+# Practicalities
+
+- Agenda, instructions and Q&A visible in collaborative HedgeDoc: [https://siili.rahtiapp.fi/kielipankki-training](https://siili.rahtiapp.fi/kielipankki-training)
+- "Lectures" and demonstrations in main Zoom room
```

Comment:
The quotes around Lectures might be too self-deprecating / confusing, as Martin said about some other slideset. Just lectures, or maybe presentations?

Reply:
Oh, presentations, that's great. I was looking for a word but not-quite-lectures was the best I came up with, but that's definitely a lot better.